static/js/models/cohortdata.model.js

```diff
@@ -130,10 +130,19 @@ Cohortdata.getCohorts = (studyID, trait, result) => {
     })
 }
 
-Cohortdata.getSummaryResults = (studyID, trait, cohort, result) => {
+Cohortdata.getSummaryResults = (studyID, trait, cohorts, result) => {
+    sqlStatement = ""
+    queryParams = []
+    if (!Array.isArray(cohorts)) {
```

Comment:
Also consider doing null check. If cohorts is null, but is put into an array, this could create problems.